README.md

```diff
@@ -13,17 +13,18 @@ adhocracy+ is designed to make online participation easy and accessible to every
 
 ### Requirements
 
- * nodejs (+ npm)
+ * nodejs (+ npm) (Use Homebrew to reinstall the node in the projects folder runnign: brew install node)
  * python 3.x (+ venv + pip)
  * libpq (only if postgres should be used)
  * sqlite3 [with JSON1 enabled](https://code.djangoproject.com/wiki/JSON1Extension)
  * redis (in production, not needed for development)
+ * pillow-heif (for macOS M1 Monterey and newer versions)
```

Comment:
did you research this is required only for M1 and newer macos? Also good to add why is it needed as the name doesn't indicate much. Was it for the python magick package which works with file types?

Reply:
hi @m4ra, thanks for your note! I did added some resources to the [issue](https://github.com/liqd/adhocracy-plus/issues/2743). I only could find out that for macOS Monterey and later versions, you need to install that from the source. Pillow-heif should help in handling HEIF/HEIC images. Should that be added?

Reply:
ok thanks, just add inside the brackets `(required for macos M1 ...)`